src/unk_02054884.c

```diff
@@ -161,22 +152,20 @@ Pokemon * sub_02054A74 (const Party * param0)
 
 BOOL Party_HasTwoAliveMons (const Party *party)
 {
-    return sub_02054A10(party) >= 2;
+    return Party_AliveMonsCount(party) >= 2;
 }
 
-void sub_02054AC4 (Party * param0)
+void Party_SetChampionRibbons (Party *party)
 {
-    int v0, v1;
-    u8 v2 = 1;
-    Pokemon * v3;
-
-    v1 = Party_GetCurrentCount(param0);
+    int i;
+    u8 ribbonActive = 1;
```

Comment:
```suggestion
    u8 championRibbon = TRUE;
```

Reply:
The fact that this doesn't match with a `BOOL` instead of `u8` makes me sad.

Reply:
Memory sizes haha!

We should still make use of the `BOOL` constants where possible, even if the underlying type is not actually `BOOL` (read: `int`).